### PR TITLE
Update command to fix user properties

### PR DESCRIPTION
**Pull request by millerdev**
@orangejenny @emord cc @nickpell

### CODE REVIEW

File: corehq/apps/app_manager/management/commands/migrate_app_to_cmitfb.py [hunk 1]
```diff
@@ -99,6 +106,57 @@ def migrate_app(self, app_id):
...
+                xform = XForm(form.source)
+                refs = {xform.resolve_path(ref): value[0]
+                    for ref, value in form.case_references.load.iteritems()
+                    if len(value) == 1 and value[0] and value[0].startswith("#user/")}
+                for node in xform.model_node.findall("{f}setvalue"):
+                    if (node.attrib.get('ref') in refs
+                            and node.attrib.get('event') == "xforms-ready"
+                            and node.attrib.get('value')
+                            and node.attrib.get('value')
+                                .startswith(BAD_USERCASE_PATH)):
```
Comment (dimagimon) [inline corehq/apps/app_manager/management/commands/migrate_app_to_cmitfb.py]: E127 continuation line over-indented for visual indent

File: corehq/apps/app_manager/management/commands/migrate_app_to_cmitfb.py [hunk 2]
```diff
@@ -99,6 +106,60 @@ def migrate_app(self, app_id):
...
 
+    def fix_user_properties(self, app_id):
+        app = Application.get(app_id)
+        if not should_fix_user_props(app):
+            return False
+        logger.info('fixing app %s', app_id)
+        modules = [m for m in app.modules if m.module_type == 'basic']
+        updated = False
+        for module in modules:
+            forms = [f for f in module.forms if f.doc_type == 'Form']
```
Comment (gcapalbo) [inline corehq/apps/app_manager/management/commands/migrate_app_to_cmitfb.py]: i don't have a lot of context here, but do you need to consider other subclasses of FormBase, or are you explicitly filtering those out

**Review by emord — APPROVED** [review]

File: corehq/apps/app_manager/management/commands/migrate_app_to_cmitfb.py (hunk 2 above)
Comment (millerdev) [inline corehq/apps/app_manager/management/commands/migrate_app_to_cmitfb.py]: I'm explicitly filtering those out.

**Comment by millerdev** [comment]
This has been run against staging, prod, and softlayer for all domains that were previously migrated.

**Review by nickpell — APPROVED** [review]
last two commits look good to me

File: corehq/apps/app_manager/management/commands/migrate_app_to_cmitfb.py [hunk 3]
```diff
@@ -1,5 +1,6 @@
 import logging
 from collections import defaultdict
+from datetime import datetime, timedelta
```
Comment (dimagimon) [inline corehq/apps/app_manager/management/commands/migrate_app_to_cmitfb.py]: F401 'timedelta' imported but unused

**Review by emord — APPROVED** [review]

**Comment by millerdev** [comment]
The most recent changes have been run against staging, prod, and softlayer for all domains that were previously migrated. I think this is most of what we can do in the way of automated fixes. There are still some domains that may need attention, especially if forms were saved after the original migration but before the fixes were applied.

**Pull request merged**